docs/sql/statements/select.rst

```diff
@@ -159,7 +159,7 @@ Joined relation
 A ``joined_relation`` is a relation which joins two relations together. See
 :ref:`sql_dql_joins` ::
 
-    relation { , | join_type JOIN } relation [ ON join_condition ]
+    relation { , | join_type JOIN } relation {[ ON join_condition ] | [USING (col_names)]}
```

Comment:
I think it is:
```suggestion
    relation { , | join_type JOIN } relation [ { ON join_condition  |  USING (col_names) } ]
```

`[ .. ]`for the optional indication, `{ a | b | c }` to denote different options.